guides/common/modules/proc_altering-the-privilege-elevation-method.adoc

```diff
@@ -0,0 +1,22 @@
+[id="altering-the-privilege-elevation-method_{context}"]
+= Altering the Privilege Elevation Method
+
+By default, push-based remote execution uses `sudo` to switch from the SSH user to the effective user that executes the script on your host.
+In some situations, you may require to use another method, such as `su` or `dzdo`.
+You can globally configure an alternative method to switch to the effective user in your {Project} settings.
```

Comment:
```suggestion
You can globally configure an alternative method to switch to the effective user in your {Project} settings, such as `su` or `dzdo`.
```
With this, you could actually remove line 5.

Reply:
I'd prefer it with the earlier sentence.